### ISSUE TITLE
Cannot build react-native project under xcode 7 beta

### ISSUE THREAD

**Issue by huston007** [facebook/react-native#2236]
I have installed XCode Version 7.0 beta 4 (7A165t), tried to build project under react-native@0.8.0 and got that error on `RTCDownloadTaskWrapper.m`
```
- (NSURLSessionDownloadTask *)downloadData:(NSURL *)url progressBlock:(RCTDataProgressBlock)progressBlock completionBlock:(RCTDataCompletionBlock)completionBlock
{
  //Got error here "Null passed  to a callee that requires a non-null argument"
  NSURLSessionDownloadTask *task = [_URLSession downloadTaskWithURL:url completionHandler:nil];
...
```

**Comment by huston007** [comment]
According to [docs](https://developer.apple.com/library/ios/documentation/Foundation/Reference/NSURLSession_class/#//apple_ref/occ/instm/NSURLSession/downloadTaskWithURL:completionHandler:):
> Unless you have provided a custom delegate, this parameter must not be nil, because there is no other way to retrieve the response data.

**Comment by ide** [comment]
It's been fixed in 0.9.0-rc :)

**Comment by huston007** [comment]
@ide Yep, have checked it, it compiles. Thanks
